src/storage/src/hummock/local_version/local_version_manager.rs

```diff
@@ -178,6 +186,7 @@ impl LocalVersionManager {
             buffer_tracker,
             sstable_id_manager,
             event_receiver,
+            version_update_rx,
```

Comment:
Since HummockEventHandler need to be construct after `LocalVersionManager`, we can split this function into two functions: one for construct `LocalVersionManager`, the other for `LocalVersionManagerExternalHolder` and `HummockEventHandler`.